.github/workflows/trigger-downstream-flake-update.yml

```
@@ -0,0 +1,30 @@
+name: Trigger Downstream Flake Update on Push to Main
+
+on:
+  push:
+    branches:
+      - main # Or the branch you want to track
+
+jobs:
+  trigger-downstream-repo:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - repo: 'blocksense-network/infra'
```

Let's rename this for better clarity:

```suggestion
          - downstream-repo: 'blocksense-network/infra'
```